scripts/build-s390x

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -ex
+
+cd $(dirname $0)/..
+
+source ./scripts/version.sh
+mkdir -p build/images
+
+./scripts/build-binary
+./scripts/build-image-runtime
+./scripts/package-binary
+./scripts/package-bundle
```

Comment:
is this just a copy of scripts/build? with all of these other changes is `make build` on an s390x host not sufficient?

Reply:
it is not a copy. We don't want to build window images and we don't support airgap on s390x

Reply:
Do we have a design document that captures the omission of airgap support for this architecture? That seems like an important thing to casually omit.

Reply:
What is missing here to provide air gap? I was under the impression that air gap was effectively pulling the images down locally and "side-loading" them into the server(s) or a local registry. Is there more to it than that?

Reply:
I would probably recommend
* Putting the windows bits of `./scripts/build` behind an architecture check, or perhaps modifying the various `scripts/*-windows-*` scripts to exit early on non-amd64 architectures so that they become a no-op.
* Modifying `scripts/build-images` to skip building airgap tarballs for the optional components on non-amd64 architectures.
Everything else should already work, as far as I can tell?

Reply:
Yes, that should work. Thanks!